src/aframe-streetmix-parsers.js

```diff
@@ -813,35 +808,56 @@ function processBuildings (left, right, streetWidth, showGround, length) {
 
     if (showGround) {
       var groundJSONString = JSON.stringify(streetmixParsersTested.createGroundArray(currentValue));
+      if ("[]" !== groundJSONString){
+        let mixin = streetmixParsersTested.createGroundArray(currentValue)[0].mixin;
+        repeat = length / 30;
```

Comment:
i think this to be `const repeat` or similar?

Reply:
https://github.com/3DStreet/3dstreet/pull/154/commits/722d2a958bf7621e6209a5e6cac449a7a99398c1